thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/route.js

```diff
@@ -31,5 +31,33 @@ export default Ember.Route.extend({
       allConfigGroups: fetch('/thirdeye/entity/ALERT_CONFIG').then(res => res.json()),
       allAppNames: fetch('/thirdeye/entity/APPLICATION').then(res => res.json())
     });
+  },
+
+  /**
+   * Model hook for the create alert route.
+   * @method resetController
+   * @param {Object} controller - active controller
+   * @param {Boolean} isExiting - exit status
+   * @param {Object} transition - transition obj
+   * @return {undefined}
+   */
+  resetController(controller, isExiting, transition) {
+    this._super(...arguments);
+    if (isExiting) {
+      controller.clearAll();
+    }
+  },
+
+  /**
+   * Model hook for the create alert route.
+   * @method setupController
+   * @param {Object} controller - active controller
+   * @param {Object} model - active model
+   * @return {undefined}
+   */
+  setupController(controller, model) {
+    this._super(controller, model);
+    controller.set('model', model);
```

Comment:
this step `controller.set('model', model);` should already be handled by ember. Any reason why we need to specifically call `setupController` here?

Reply:
Yes, its redundant. I'll remove it.